flixel/tweens/misc/VarTween.hx

```diff
@@ -37,6 +37,7 @@ class VarTween extends FlxTween
 		_propertyInfos = [];
 		this.duration = duration;
 		start();
+		initializeVars();
```

Comment:
Are any of the changes in `VarTween` changes besides the `isTweenOf()` override related to this PR? It's not clear to me why they were made.

Reply:
It calls initializeVars earlier than before so that we can check `_propertyInfos` in `isTweenOf()`. originally, this initialization was delayed until the tween starts so that chaining works correctly, but that's only needed for the startValues.

If you think I need to explain this in code, let me know